doc/omnipytent.txt

```diff
@@ -152,6 +154,14 @@ You can add the name of a task as an argument to these commands - for example: >
 This will open the tasks file and place the cursor on the "run" task. If the
 "run" task does not exist, it will create a basic skeleton for it.
 
+If a tasks file exists in a parent directory(or parent of parent, etc.), it
```

Comment:
Add space before `(`

Reply:
I notice that you consistently lack the space before the parantheses. I think this is a (minor) grammatical error, and I would suggest that you add a space before the opening parantheses throughout.